src/index.ts

```diff
@@ -44,7 +45,7 @@ export default {
    *   style: '.hello{color: red}',
    * })
    */
-  init(config = {}) {
+  init(config:EditorConfig) {
```

Comment:
Missing space after `:`.

```suggestion
  init(config: EditorConfig) {
```